cli/local/model_cache.go

```diff
@@ -31,8 +31,31 @@ import (
 	s "github.com/cortexlabs/cortex/pkg/lib/strings"
 	"github.com/cortexlabs/cortex/pkg/lib/zip"
 	"github.com/cortexlabs/cortex/pkg/types/spec"
+	"github.com/cortexlabs/cortex/pkg/types/userconfig"
 )
 
+func CacheModels(apiSpec *spec.API, awsClient *aws.Client) ([]*spec.LocalModelCache, error) {
+	modelPaths := make([]string, len(apiSpec.Predictor.Models))
+	for i, modelResource := range apiSpec.Predictor.Models {
+		modelPaths[i] = modelResource.Model
+	}
+
+	localModelCaches := make([]*spec.LocalModelCache, len(modelPaths))
+	var err error
```

Comment:
Doing `localModelCaches[i], err := CacheModel(modelPath, awsClient)` gets me a syntax error because `localModelCaches` is already defined. I moved the error declaration on line 46 though `var err error`.